lib/Subscriber/Core/RegistratorSubscriber.php

```diff
@@ -230,4 +211,25 @@ private function handleRegister(AbstractMappingEvent $event)
 
         $this->documentRegistry->registerDocument($document, $node, $locale);
     }
+
+    /**
+     * If the node for the persisted document is in the registry.
+     *
+     * @param PersistEvent|ReorderEvent $event
+     */
+    public function handleNodeFromRegistry($event)
```

Comment:
Why move this? `private` functions are intentionally placed below `public` functions.